### PR TITLE
Feature/grades students panel

### LINKED ISSUE #7: Oceny – panel ucznia
During implementation think about TS.

1. There should be a section with notification for every student, optional fade notifications after one minute 
2. On left side there should be a student list
3. In the middle there should be input for student to write "student secret" and get access to grades
4. After providing "student secret" there should be a grid with grades, work with Justyna, there should be on source of truth for studnets (firebase, local storage or json file. some kind of service with fake data)

### PR DESCRIPTION
**Pull request by neacatsu**

### CODE REVIEW

File: src/gradesStudentsPanel.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GradeBook</title>
+    <script src="assets/ts/gradesStudentsPanel.ts" async></script>
```
Comment (kingam280) [inline src/gradesStudentsPanel.html]: You don't have to import script here, webpack does it;)

File: src/assets/ts/gradesStudentsPanel.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        if(isSecret){
+            if(element.user_id  === id) {
+                gradesArray.forEach((el) => {              
+                    if(el.subject.toLowerCase() == element.subject) {
+                        el.grades.push(element.grade)
+                    }
+                })
+            }
+        } else {
+            outputList = "Podaj dobry secret"
```
Comment (kingam280) [inline src/assets/ts/gradesStudentsPanel.ts]: Maybe change text to english

**Review by kingam280 — COMMENTED** [review]
Looks fine to me, good job :)

One thing, in file SubPagesData.ts change name to your html file name so menu can work correctly

File: src/gradesStudentsPanel.html (hunk 1 above)
Comment (neacatsu) [reply src/gradesStudentsPanel.html]: done :)

File: src/assets/ts/gradesStudentsPanel.ts (hunk 2 above)
Comment (neacatsu) [reply src/assets/ts/gradesStudentsPanel.ts]: done :)

**Review by mateugazi — APPROVED** [review]

**Review by MaDud — APPROVED** [review]
Good job, resolve confilt and we almost have it!

**Review by JotGabiga — APPROVED** [review]

**Review by mateugazi — APPROVED** [review]

**Review by JotGabiga — APPROVED** [review]

**Pull request merged**